spaces/tokens.json

```
@@ -152,5 +152,7 @@
   "0xeaccb6E0f24d66cF4Aa6cBDa33971b9231d332a1": "polyient-games-governance",
   "0xaea8e1b6cb5c05d1dac618551c76bcd578ea3524": "sogur",
   "0x683EcFe5B148c8840998c1B4884F60a8eAD432D1": "d5",
-  "0xa150db9b1fa65b44799d4dd949d922c0a33ee606": "digital-reserve-currency"
+  "0xa150db9b1fa65b44799d4dd949d922c0a33ee606": "digital-reserve-currency",
+  "0x13748d548d95d78a3c83fe3f32604b4796cffa23": "kogecoin"
+
```

Please do not add your token to this file, you don't need it.
```suggestion
  "0xa150db9b1fa65b44799d4dd949d922c0a33ee606": "digital-reserve-currency"
```

---

Removed from tokens.json.